src/components/widgets/Mo2PercentWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {IPoint} from 'types/IPoint';
+import {BarChart} from "components/widgets/shared/BasicBarGraph";
+import {XYPlotTemplate} from 'components/widgets/shared/XYPlotTemplate';
+import {
+  BasicHorizontalAxis,
+  BasicVerticalAxis
+} from 'components/widgets/shared/BasicAxes';
+
+const MIN_POINTS_BEFORE_SHOW = 6;
+
+export interface IMo2PercentProps {
```

Comment:
I believe I created an interface that you can extend here. Look at the heart and breathe fate widget

Reply:
I think its more it'll be more consistent if each widget had its own interface

Reply:
That’s fair. I didn’t even follow my own convention in the heart rate widget... the core body temp has the convention I would like to follow. We need widthPx and heightPx on every graph so I made an interface for it